### PR TITLE
Unit tests for cordova 2.0.0 bridge

### PR DESCRIPTION
**Pull request by mdineen**
Cleaned up changes backfilling cordova 2.0.0 bridge unit tests.

### CODE REVIEW

**Comment by brentlintner** [comment]
@gtanner If you get a chance, mind giving this a +1 from yourself (and it is Cordova stuff, to boot)? I've gone over this earlier today (not too in depth, though).

File: test/unit/cordova/capture.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            capture = require('ripple/platform/cordova/2.0.0/bridge/capture');
+            event = require('ripple/event');
+            s = jasmine.createSpy("success");
+            e = jasmine.createSpy("error");
+            captureSpy = jasmine.createSpy("captured-image");
+            event.once("captured-image", captureSpy);
+            spyOn(camera, "show");
+        });
+
+        xit("can be called with no args", function () {
```
Comment (brentlintner) [inline test/unit/cordova/capture.js:41]: @mdineen was this still failing? Not sure if the `xit` was missed.

Comment (mdineen) [inline test/unit/cordova/capture.js:41]: Yes, still failing.

File: test/unit/cordova/console.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    describe("when logging", function () {
+        it("calls the log method", function () {
+            spyOn(console, "log");
+            _console.log(null, null, ["beavers!!!"]);
+            expect(console.log).toHaveBeenCalledWith("beavers!!!");
+        });
+    });
+
+
```
Comment (brentlintner) [inline test/unit/cordova/console.js]: Empty lines _could_ be removed (nitpick, lol). :-P

File: test/unit/cordova/contacts.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    }, {
+        "name": { formatted: "Gord Tanner" },
+        "displayName": "Gord Tanner",
+        "emails": [{ type: "work", value: "gord@tinyhippos.com", pref: true }]
+    }, {
+        "name": { formatted: "Mark McArdle" },
+        "displayName": "Mark McArdle",
+        "emails": [{ type: "work", value: "mark@tinyhippos.com", pref: false }]
+    }],
+    navi;
```
Comment (brentlintner) [inline test/unit/cordova/contacts.js]: Usually, var statements with multiple lines are indented once more to the right (IMO better readability), but your call, that is an opinion, and a nitpick. :-)

Comment (mdineen) [inline test/unit/cordova/contacts.js]: Done

File: test/unit/cordova/console.js (hunk 2 above)
Comment (mdineen) [inline test/unit/cordova/console.js]: Done

**Pull request merged**